toolkit/scripts/srpm_pack.mk

```diff
@@ -14,12 +14,22 @@
 SRPM_FILE_SIGNATURE_HANDLING ?= enforce
 
 SRPM_BUILD_CHROOT_DIR = $(BUILD_DIR)/SRPM_packaging
+SRPM_BUILD_LOGS_DIR = $(LOGS_DIR)/pkggen/srpms
 
+toolchain_spec_list = $(toolchain_build_dir)/toolchain_specs.txt
```

Comment:
Is this still used?
```suggestion
```

Reply:
Yes, it's used when packing the toolchain specs outside of a chroot environment- see `build_toolchain_srpms.flag` target when `DOWNLOAD_SRPMS=n`. I just moved it around to be with the other pack list.